code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -259,9 +259,10 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 		if(amount > 0)
 			to_chat(summoner, "<span class='danger'><B>Your [name] is under attack! You take damage!</span></B>")
 			summoner.visible_message("<span class='danger'><B>Blood sprays from [summoner] as [src] takes damage!</B></span>")
-			if(summoner.stat == UNCONSCIOUS)
-				to_chat(summoner, "<span class='danger'><B>Your body can't take the strain of sustaining [src] in this condition, it begins to fall apart!</span></B>")
-				summoner.adjustCloneLoss(amount * 0.5) //dying hosts take 50% bonus damage as cloneloss
+			switch(summoner.stat)
```

Comment:
is using a switch actually useful here?

Reply:
It does two checks in one, and supports more easily adding further cases. Most `stat` checks benefit from this method because of the multiple values it can hold.

Reply:
that sounds fair enough then